bpf/cpu/cpu.bpf.c

```diff
@@ -747,9 +747,18 @@ int walk_user_stacktrace_impl(struct bpf_perf_event_data *ctx) {
         return 1;
       }
 
-      // TODO(sylfrena): add comments to explain calculations
+      // Stacktraces are saved return addresses from function calls pushed onto a stack
+      // The base pointer(bp) is first register pushed onto the stack and holds the beginning of the stack
+      // Then, the stack pointer points at the first frame at the bp updating the top of the stack to 8 bits ahead of the bp
```

Comment:
8 bytes no? (and same below)

Reply:
oh no, you're right! good catch, thanks!